<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Creation_Client</class>
 <widget class="QMainWindow" name="Creation_Client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1039</width>
    <height>858</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Creation Client</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="usesScrollButtons">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Nouveau Client</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QFormLayout" name="formLayout">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <property name="horizontalSpacing">
           <number>6</number>
          </property>
          <property name="verticalSpacing">
           <number>16</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>2</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Nom Complet</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdit_client_nom">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Abreviation</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEdit__client_abrev"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Adresse 
siege social</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QTextEdit" name="textEdit_client_adresse"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Code Postal</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="lineEdit__client_code_p"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_5">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Ville</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="lineEdit__client_ville"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_6">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>N° Telephone</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="lineEdit__client_tel"/>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_7">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>N° Fax</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLineEdit" name="lineEdit__client_fax"/>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_8">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Courriel</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QLineEdit" name="lineEdit__client_courriel"/>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="label_9">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Contact</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QLineEdit" name="lineEdit__client_contact"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Sites</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QFormLayout" name="formLayout_2">
            <property name="fieldGrowthPolicy">
             <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
            </property>
            <property name="horizontalSpacing">
             <number>6</number>
            </property>
            <property name="verticalSpacing">
             <number>16</number>
            </property>
            <item row="2" column="0">
             <widget class="QLabel" name="label_10">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>2</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <family>Calibri</family>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Nom Complet</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="lineEdit__site_nom">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_11">
              <property name="font">
               <font>
                <family>Calibri</family>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Abreviation</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="lineEdit__site_abrev"/>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_12">
              <property name="font">
               <font>
                <family>Calibri</family>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Adresse</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label_13">
              <property name="font">
               <font>
                <family>Calibri</family>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Code Postal</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QLineEdit" name="lineEdit__site_code_p"/>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="label_14">
              <property name="font">
               <font>
                <family>Calibri</family>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Ville</string>
              </property>
             </widget>
            </item>
            <item row="6" column="1">
             <widget class="QLineEdit" name="lineEdit__site_ville"/>
            </item>
            <item row="7" column="0">
             <widget class="QLabel" name="label_15">
              <property name="font">
               <font>
                <family>Calibri</family>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>N° Telephone</string>
              </property>
             </widget>
            </item>
            <item row="7" column="1">
             <widget class="QLineEdit" name="lineEdit__site_tel"/>
            </item>
            <item row="8" column="0">
             <widget class="QLabel" name="label_16">
              <property name="font">
               <font>
                <family>Calibri</family>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>N° Fax</string>
              </property>
             </widget>
            </item>
            <item row="8" column="1">
             <widget class="QLineEdit" name="lineEdit__site_fax"/>
            </item>
            <item row="9" column="0">
             <widget class="QLabel" name="label_17">
              <property name="font">
               <font>
                <family>Calibri</family>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Courriel</string>
              </property>
             </widget>
            </item>
            <item row="9" column="1">
             <widget class="QLineEdit" name="lineEdit__site_courriel"/>
            </item>
            <item row="10" column="0">
             <widget class="QLabel" name="label_18">
              <property name="font">
               <font>
                <family>Calibri</family>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Contact</string>
              </property>
             </widget>
            </item>
            <item row="10" column="1">
             <widget class="QLineEdit" name="lineEdit__site_contact"/>
            </item>
            <item row="4" column="1">
             <widget class="QTextEdit" name="textEdit___site_adresse"/>
            </item>
            <item row="0" column="0">
             <widget class="QCheckBox" name="checkBox_2">
              <property name="text">
               <string>Site EFS</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="comboBox_poste_tech_sap_efs">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="editable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_22">
              <property name="text">
               <string>Prefixe SAP</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="lineEdit_prefixe_sap">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_sites_ajouter">
            <property name="text">
             <string>Ajouter</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_maj">
            <property name="text">
             <string>Mise à jour</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Tablewidget_Recap_Site" name="tableWidget__site_recap">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>10</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <column>
             <property name="text">
              <string>Nom Complet</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Abreviation</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Adresse</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Code Postal</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Ville</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>N°Telephone</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>N° Fax</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Courriel</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Contact</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Prefixe_SAP</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Services</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_21">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <family>Calibri</family>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Sites</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_sites">
              <item>
               <property name="text">
                <string>*</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout_3">
            <property name="fieldGrowthPolicy">
             <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
            </property>
            <property name="horizontalSpacing">
             <number>6</number>
            </property>
            <property name="verticalSpacing">
             <number>16</number>
            </property>
            <item row="1" column="0">
             <widget class="QLabel" name="label_19">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>2</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <family>Calibri</family>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Nom Complet</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="lineEdit_service_nom">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_20">
              <property name="font">
               <font>
                <family>Calibri</family>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Abreviation</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="lineEdit_service_abrev"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_24">
              <property name="font">
               <font>
                <family>Calibri</family>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>N° Telephone</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="lineEdit_service_tel"/>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_25">
              <property name="font">
               <font>
                <family>Calibri</family>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>N° Fax</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLineEdit" name="lineEdit_service_fax"/>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label_26">
              <property name="font">
               <font>
                <family>Calibri</family>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Courriel</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QLineEdit" name="lineEdit_service_courriel"/>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="label_27">
              <property name="font">
               <font>
                <family>Calibri</family>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Contact</string>
              </property>
             </widget>
            </item>
            <item row="6" column="1">
             <widget class="QLineEdit" name="lineEdit_service_contact"/>
            </item>
            <item row="0" column="0">
             <widget class="QCheckBox" name="checkBox">
              <property name="text">
               <string>Services EFS</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="comboBox_service_efs">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="editable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_service_ajouter">
            <property name="text">
             <string>Ajouter</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_service_maj">
            <property name="text">
             <string>Mise à jour</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Tablewidget_Recap_Service" name="tableWidget__recap_service">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>10</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <column>
             <property name="text">
              <string>Sites</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Nom Complet</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Abreviation</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>N°Telephone</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>N° Fax</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Courriel</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Contact</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::NoToolBarArea</set>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionSauvegarder"/>
  </widget>
  <action name="actionSauvegarder">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icones.qrc">
     <normaloff>:/Sauvegarde/b_1_q_0_p_0.jpg</normaloff>:/Sauvegarde/b_1_q_0_p_0.jpg</iconset>
   </property>
   <property name="text">
    <string>Sauvegarder</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Tablewidget_Recap_Site</class>
   <extends>QTableWidget</extends>
   <header>GUI.Clients.Tablewidget_recap_site</header>
  </customwidget>
  <customwidget>
   <class>Tablewidget_Recap_Service</class>
   <extends>QTableWidget</extends>
   <header>GUI.Clients.Tablewidget_recap_service</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icones.qrc"/>
 </resources>
 <connections/>
</ui>
